integration-tests/pkgJson-restore.spec.js

```diff
@@ -14,1449 +14,681 @@
     specific language governing permissions and limitations
     under the License.
 */
-var helpers = require('../spec/helpers');
-var path = require('path');
-var shell = require('shelljs');
-var events = require('cordova-common').events;
-var ConfigParser = require('cordova-common').ConfigParser;
-var cordovaPlatform = require('../src/cordova/platform');
-var prepare = require('../src/cordova/prepare');
-var cordova_util = require('../src/cordova/util');
-var cordovaPlugin = require('../src/cordova/plugin');
-// cordova = require('../src/cordova/cordova'),
-var TIMEOUT = 240 * 1000;
+
+const path = require('path');
+const fs = require('fs-extra');
+const prepare = require('../src/cordova/prepare');
+const cordovaPlugin = require('../src/cordova/plugin');
+const cordovaPlatform = require('../src/cordova/platform');
+const { ConfigParser } = require('cordova-common');
+const { listPlatforms, requireNoCache } = require('../src/cordova/util');
+const { tmpDir: getTmpDir, testPlatform, setDefaultTimeout } = require('../spec/helpers');
 
 /** Testing will check if "cordova prepare" is restoring platforms and plugins as expected.
 *   Uses different basePkgJson files depending on testing expecations of what (platforms/plugins/variables)
 *   should initially be in pkg.json and/or config.xml.
 */
+describe('restore', function () {
+    setDefaultTimeout(240 * 1000);
```

Comment:
minor nit: I would rather see this defined as a constant at the beginning, even if it now appears only once.

Reply:
I actually like this better the way it is. The function name already tells us what the argument is. So it's not really a magic number.

Something like `setDefaultTimeout(4 * ONE_MINUTE_IN_MS)` would be nicer, but hopefully we will be able to reduce this soon anyway.

That being said, I have no strong feelings about this. So I will change it and merge this part of the PR.